herald/generate-player-data.js

```diff
@@ -133,14 +132,40 @@ let hibernia =
     ],
 }
 
-let realms = [albion, midgard, hibernia];
+//let realms = [albion, midgard, hibernia];
+
+let realms = {
+    albion: albion,
+    midgard: midgard,
+    hibernia: hibernia,
+}
 
 let players = [];
 let playerNames = [];
+let playersByRealm = {
+    albion: [],
+    midgard: [],
+    hibernia: [],
+};
+
+//we want a player to always be present
+players.push({
+        fullName: "Testplayer",
+        raceName: "briton",
+        className: "cleric",
+        level: 10,
+        xpPercent: 1,
+        rpPercent: 1,
+        realmRank: 1,
+        realm: 'albion',
+});
+
+let playerNames.push('Testplayer');
```

Comment:
Well that's the thing, it didn't work for me.  It errored with a SyntaxError at runtime.
```
\herald\generate-player-data.js:163
let playerNames.push('Testplayer');
    ^^^^^^^^^^^
SyntaxError: Identifier 'playerNames' has already been declared
    at Object.exports.runInThisContext (vm.js:78:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.runMain (module.js:605:10)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:535:3
```
Thanks for fixing it!